gulpfile.js

```diff
@@ -23,15 +23,15 @@ gulp.task('browser', function() {
     .pipe(gulp.dest('./Example/source'));
 });
 
-gulp.task('es6', function(){
+gulp.task('es6',_ => {
```

Comment:
@ManrajGrover This is because in `browser` task of gulp you require `./Example/test.js` but but in your `.gitignore` you have the same file blacklisted from uploading to git i dont know why.

That is the reason i can see from my terminal

![image](https://cloud.githubusercontent.com/assets/10435209/16183843/2c5b209c-36d4-11e6-8758-0ad469e30379.png)

Please verify.